src/docs/implementation/DataPLAN.md

```diff
@@ -11,45 +11,48 @@ article_status: published
 
-<figure><img src="../img/dataplan_after_deselect.png" alt="After deselect checkboxes" style="width:100%"><figcaption style="margin: auto;">Figure 3: If some of the checkboxes are deselected in quesiton 2, the respective counterparts in the DMP text will be removed.</figcaption></figure>  
 
-For specific information which is unique between different project, such as the "project name" and the "project aim", the users can can type text in the respective input fields. The counter parts in the DMP will be updated lively as well.   
+<br> For specific information, which is unique between different projects, such as the "project name" and the "project aim", the users can use free-text for the respective input fields. The counter parts in the DMP will be updated lively as well.   
 
-## The templates can be customized
-All the templates prepared and provided by the DataPLAN are customizable. DataPLAN has a user-defined template section (Fig. 4) where the template are editable and can be directly loaded into the DataPLAN.   
+## Templates can be customized
+All templates prepared and provided by DataPLAN are customizable. DataPLAN has a user-defined template section (Fig. 4), in which templates are editable and can be directly loaded into DataPLAN.   
 
-<figure><img src="../img/dataplan_user_defined.png" alt="User-defined template can also be directly used" style="width:100%"><figcaption style="margin: auto;">Figure 4: All the templates prepared and provided by the DataPLAN are Customizable. The templates can be reused to create user-defined templates.</figcaption></figure>
+<figure><img src="../img/dataplan_user_defined.png" alt="User-defined template can also be directly used" style="width:100%"><figcaption style="margin: auto;">Figure 4: All  templates prepared and provided by the DataPLAN are customizable. The templates can be reused to create user-defined templates.</figcaption></figure>
 
 ## Offline usable and absolute secure
-DataPLAN can be used offline conveniently. Unlike other DMP tools, which is complicated to run in offline situations, the DataPLAN can be saved as html file (by "ctrl + s" as showed in Fig. 5) and then be further used. The user can also disconnect the internet after fully load the DataPLAN website. The server-less design of DataPLAN make it absolute secure, because it prevent the loss and exposure of data during either internet data transfer or online storage.  
+DataPLAN can be used offline conveniently. Unlike other DMP tools, which can be complicated to run in offline situations, the document can be saved as html file using "ctrl + s" (Fig. 5) and edited subsequently. The user can also disconnect the internet after the DataPLAN website has been loaded completely. The serverless design of DataPLAN makes it absolute secure, because it prevents the loss and exposure of data during internet data transfer or online storage.  
```

Comment:
Here I mean the "DataPLAN" itself can be saved and further used , not only the document but the entire tool.